app/Field/BooleanFieldView.js

```diff
@@ -0,0 +1,21 @@
+class BooleanFieldView {
+    static getReadWidget() {
+        return '<BooleanColumn value={this.props.value} />';
```

Comment:
I don't understand why you return a string and not a JSX. You could avoid compile altogether by using JSX. And it's more React-y.